com.unity.render-pipelines.high-definition/Documentation~/Default-Settings-Window.md

```diff
@@ -9,50 +9,50 @@ The High Definition Render Pipeline (HDRP) adds the HDRP Settings tab to Unity's
 | Lens Attenuation Mode                 | Set the attenuation mode of the lens that is used to compute exposure. With imperfect lens some energy is lost when converting from EV100 to the exposure multiplier, while a perfect lens has no attenuation and no energy is lost. |
 | Use DLSS Custom Project Id            | Controls whether to use a custom project ID for the NVIDIA Deep Learning Super Sampling module. If you enable this property, you can use **DLSS Custom Project Id** to specify a custom project ID.<br/>This property only appears if you enable the NVIDIA package (com.unity.modules.nvidia) in your Unity project. |
 | DLSS Custom Project Id                | Controls whether to use a custom project ID for the NVIDIA Deep Learning Super Sampling (DLSS) module. If you enable this property, you can use **DLSS Custom Project Id** to specify a custom project ID. If you disable this property, Unity generates a unique project ID. <br/>This property only appears if you enable the NVIDIA package (com.unity.modules.nvidia) in your Unity project. |
+| Runtime Debug Shaders                 | When disabled, all debug display shader variants are removed when you build for the Unity Player. This decreases build time, but prevents the use of Rendering Debugger in Player builds. |
+
+## Resources
+
+Resources assets list the Shaders, Materials, Textures, and other Assets needed to operate the High Definition Render Pipeline.
+
+| **Property**              | **Description**                                              |
+| ------------------------- | ------------------------------------------------------------ |
+| **Player Resources**      | Stores references to Shaders and Materials that HDRP uses. When you build your Unity Project, HDRP embeds all of the resources that this Asset references. It allows you to set up multiple render pipelines in a Unity Project and, when you build the Project, Unity only embeds Shaders and Materials relevant for that pipeline. This is the Scriptable Render Pipeline equivalent of Unity’s Resources folder mechanism. When you create a new HDRP Global Settings Asset, the HDRenderPipelineRuntimeResources from HDRP package is automatically referenced in it. |
+| **Ray Tracing Resources** | Stores references to Shaders and Materials that HDRP uses for ray tracing. HDRP stores these resources in a separate Asset file then the main pipeline resources so it can use less memory for applications that don't support ray tracing. When you create a new HDRP Global Settings Asset, the HDRenderPipelineRayTracingResources from HDRP package is automatically referenced in it if your project use ray tracing. |
```

Comment:
Replace "then" with "from": 

"...HDRP stores these resources in a separate Asset file from the main pipeline resources so that it can use less memory for applications that don't support ray tracing..."

Reply:
@Vic-Cooper thanks for looking over these! Just to clarify what I've changed here - I've only added documentation that relates to HDRP/URP Global Settings Assets. All other changes are simply reorganizing the documentation so that they appear in the same order as the settings UI itself, to make it easier for people to read; sadly the diffs don't make this easy to see. I'm happy to make the all of the suggested changes if you prefer, but just wanted to clarify that this PR didn't add them :)

Reply:
That's alright! As I take over docs I haven't touched before I might see the occasional thing I'd like to tweak (a few typos and things). 
The content here is otherwise great :)